Makefile

```diff
@@ -3,7 +3,7 @@
 all: ps erl
 
 ps:
-	psc-package sources | xargs purs compile 'test/**/*.purs' 'src/**/*.purs'
+	spago build
```

Comment:
I took a look at how it was done in Pinto:
```
output: $(PS_SOURCEFILES) $(PS_ERL_FFI) .psc-package
	.psc-package/${PACKAGE_SET}/erl-modules/${ERL_MODULES_VERSION}/scripts/gen_module_names.sh src/Pinto Pinto.ModuleNames
	psc-package sources | xargs purs compile '$(PS_SRC)/**/*.purs'
	@touch output
```
which pretty much looks like what spago would do under the hood only problem when calling `make test` it falls over on from what I can make out because of the exporting of a module.